src/Compatibility/Microsoft.DotNet.ApiCompatibility/Logging/SuppressionEngine.cs

```diff
@@ -55,24 +56,32 @@ public bool IsErrorSuppressed(string diagnosticId, string? target, string? left
         }
 
         /// <summary>
-        /// Checks if the passed in error is suppressed or not.
+        /// Checks if the passed in error is suppressed.
         /// </summary>
         /// <param name="error">The <see cref="Suppression"/> error to check.</param>
         /// <returns><see langword="true"/> if the error is already suppressed. <see langword="false"/> otherwise.</returns>
         public bool IsErrorSuppressed(Suppression error)
         {
+            // An error is suppressed if
```

Comment:
I suppose this incomplete comment should be removed, as I'm guessing the comment bellow this replaced it